meson.build

```diff
@@ -29,4 +29,76 @@ polkit_gobject = dependency('polkit-gobject-1')
 polkitpolicydir = polkit_gobject.get_pkgconfig_variable('policydir',
   define_variable: ['prefix', prefix])
 
-subdir('accounts-service')
\ No newline at end of file
+config_h = configuration_data()
+config_h.set_quoted('GETTEXT_PACKAGE', meson.project_name())
+configure_file(
+  output: 'config.h',
+  configuration: config_h,
+)
+root_inc = include_directories('.')
+
+# Enable warning flags
+test_c_args = [
+  '-fno-strict-aliasing',
+  '-fstack-protector-strong',
+  '-Waggregate-return',
+  '-Wall',
+  '-Wunused',
+  '-Warray-bounds',
+  '-Wcast-align',
+  '-Wclobbered',
+  '-Wno-declaration-after-statement',
+  '-Wduplicated-branches',
+  '-Wduplicated-cond',
+  '-Wempty-body',
+  '-Wformat=2',
+  '-Wformat-nonliteral',
+  '-Wformat-security',
+  '-Wformat-signedness',
+  '-Wignored-qualifiers',
+  '-Wimplicit-function-declaration',
+  '-Wincompatible-pointer-types',
+  '-Wincompatible-pointer-types-discards-qualifiers',
+  '-Winit-self',
+  '-Wint-conversion',
+  '-Wlogical-op',
+  '-Wmisleading-indentation',
+  '-Wmissing-declarations',
+  '-Wmissing-format-attribute',
+  '-Wmissing-include-dirs',
+  '-Wmissing-noreturn',
+  '-Wmissing-parameter-type',
+  '-Wmissing-prototypes',
+  '-Wnested-externs',
+  '-Wno-error=cpp',
+  '-Wno-discarded-qualifiers',
+  '-Wno-missing-field-initializers',
+  '-Wno-suggest-attribute=format',
+  '-Wno-unused-parameter',
+  '-Wnull-dereference',
+  '-Wold-style-definition',
+  '-Woverflow',
+  '-Woverride-init',
+  '-Wparentheses',
+  '-Wpointer-arith',
+  '-Wredundant-decls',
+  '-Wreturn-type',
+  '-Wshadow',
+  '-Wsign-compare',
+  '-Wstrict-aliasing=2',
+  '-Wstrict-prototypes',
+  '-Wswitch-default',
+  '-Wswitch-enum',
+  '-Wtype-limits',
+  '-Wundef',
+  '-Wuninitialized',
+  '-Wunused-but-set-variable',
+  '-Wunused-result',
+  '-Wunused-variable',
+  '-Wwrite-strings'
+]
```

Comment:
I think it's easier to just use `-Wall -Wextra -Weverything` if we want all those warnings.

Reply:
Not sure `-Weverything` gives the right curated set of warnings. Not fussed enough about going back into that argument to change this now.